src/core/api/public_api.ts

```diff
@@ -1338,7 +1338,16 @@ class Player extends EventEmitter<IPublicAPIEvent> {
 
     const { isDirectFile, manifest } = this._priv_contentInfos;
     if (isDirectFile) {
-      return this.videoElement.currentTime;
+      const mediaElement : HTMLMediaElement & {
+        getStartDate? : () => number | null | undefined;
+      } = this.videoElement;
+      if (typeof mediaElement.getStartDate === "function") {
```

Comment:
Maybe adding a comment explaining that this is specific to Safari, or where it's used so we don't forget it

Reply:
:+1: Will do

Reply:
done